core/src/components/item/test/buttons/index.html

```diff
@@ -188,6 +347,11 @@ <h3>Avatar Start</h3>
     </style>
 
     <script>
+      const params = new URL(window.location).searchParams;
+      const dark = params.get('dark');
+      if (dark !== null) {
+        document.body.classList.add('dark');
+      }
```

Comment:
Once we move to the generators + with Brandy's proposed dark mode testing improvement this should be no longer needed.

Reply:
Can we make a tech debt ticket for this dependent on FW-1774?